judge/views/api/api_v2.py

```diff
@@ -536,17 +568,27 @@ def get_object_data(self, profile):
                 contest__in=Contest.get_visible_contests(self.request.user),
                 contest__end_time__lt=self._now,
             )
+            .annotate(
+                old_rating=Subquery(old_ratings_subquery.values('rating')[:1]),
+                new_rating=Subquery(new_ratings_subquery.values('rating')[:1]),
```

Comment:
Not sure if this gets optimized properly, but why not just use `rating__rating` and `rating__volatility` instead of a subquery here?

Reply:
That's true, could the same reasoning be applied to https://github.com/JoshuaTianYangLiu/online-judge/blob/api-consistency/judge/views/api/api_v2.py#L255?

Reply:
Yes